Makefile

```diff
@@ -36,15 +36,13 @@ bench:
 	$(info INFO: Starting build $@)
 	go test -bench=.
 
-alpha-test-%: release/goss-%
+test-int-validate-%: release/goss-%
```

Comment:
I could equally have called this `test-int-all-commands-%` and similarly `run-command-tests.sh` but I thought that might have been confusing with the `command` assertion specifically; what do you think, @aelsabbahy?

Reply:
no opinion on this currently.

I have some plans to re-write all of the integration tests.. but want to wait until after `v4`